src/theory/ite_utilities.h

```diff
@@ -240,14 +241,8 @@ class ITESimplifier {
   uint32_t d_citeEqConstApplications;
 
   typedef std::pair<Node, Node> NodePair;
-  struct NodePairHashFunction {
-    size_t operator () (const NodePair& pair) const {
-      size_t hash = 0;
-      hash = 0x9e3779b9 + NodeHashFunction().operator()(pair.first);
-      hash ^= 0x9e3779b9 + NodeHashFunction().operator()(pair.second) + (hash << 6) + (hash >> 2);
-      return hash;
-    }
-  };/* struct ITESimplifier::NodePairHashFunction */
+  typedef PairHashFunction<Node, Node, NodeHashFunction, NodeHashFunction>
+      NodePairHashFunction;
```

Comment:
nitpick: Switch to "using NodePairHashFunction = PairHashFunction<Node, Node, NodeHashFunction, NodeHashFunction>;"?

Reply:
Nice! I didn't know about that syntax. That is actually much more readable than `typedef` :)